java/src/main/java/com/saucelabs/saucebindings/configs/SauceConfigsChrome.java

```diff
@@ -0,0 +1,34 @@
+package com.saucelabs.saucebindings.configs;
+
+import com.saucelabs.saucebindings.SauceOptions;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class SauceConfigsChrome extends SauceConfigsBrowser<SauceConfigsChrome> {
+    /**
+     * @deprecated Not for public use
+     *      Use: {@link SauceOptions#chrome()}} or {@link SauceOptions#chrome(ChromeOptions)}}
+     */
+    @Deprecated
```

Comment:
@diemol I don't understand what the impact of that is. The idea of deprecating it is so that people can still use it but know they need to update their behavior before the next major release. What does changing the package visibility do?

Reply:
Ok, I moved SauceOptions into a new package, which I think is *bad. I think it is ok if we release this as a 2.0?